org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/ITextRegionDiffBuilder.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017 TypeFox GmbH (http://www.typefox.io) and others.
+ * Copyright (c) 2017, 2019 TypeFox GmbH (http://www.typefox.io) and others.
```

Comment:
We have 2020 now I guess

Reply:
You are absolutely right, I updated the PR.